utils/util_carto.py

```diff
@@ -14,21 +14,40 @@
 CARTO_USER = os.getenv('CARTO_WRI_RW_USER')
 CARTO_KEY = os.getenv('CARTO_WRI_RW_KEY')
 
-def upload_to_carto(file, privacy, collision_strategy='skip'):
+def upload_to_carto(file, privacy, *tags:str, collision_strategy='skip'):
     '''
     Upload tables to Carto
     INPUT   file: location of file on local computer that you want to upload (string)
             privacy: the privacy setting of the dataset to upload to Carto (string)
+            tags: add one or more tags(string) to the dataset, default is "rw"
             collision_strategy: determines what happens if a table with the same name already exists
+            due to changes in carto api if set to skip and table name already exists it will raise an exception
             set the parameter to 'overwrite' if you want to overwrite the existing table on Carto
     '''
     # set up carto authentication using local variables for username (CARTO_WRI_RW_USER) and API key (CARTO_WRI_RW_KEY)
     auth_client = APIKeyAuthClient(api_key=CARTO_KEY, base_url="https://{user}.carto.com/".format(user=CARTO_USER))
     # set up dataset manager with authentication
     dataset_manager = DatasetManager(auth_client)
     # upload dataset to carto
-    dataset = dataset_manager.create(file, collision_strategy = collision_strategy)
-    logger.info('Carto table created: {}'.format(os.path.basename(file).split('.')[0]))
+    try:
+        dataset = dataset_manager.create(file, collision_strategy = collision_strategy)
+        logger.info('Carto table created: {}'.format(os.path.basename(file).split('.')[0]))
+    except ValueError:
+        logging.exception("Table probably already existing, try setting collision_strategy to overwrite or using another name")
+        # If exception is raised terminate program
+        sys.exit(1)
+    # adding tags
+    # pass one or more tags as strings, default is "rw"
+    if tags:
+        new_tags = []
+        for tag in tags:
+            new_tags.append(tag)
+    else:
+        new_tags = ['rw']
+    # set up tags     
+    dataset.tags = new_tags
+    for tag in new_tags:
+        logger.info('Adding the following tag to table:{}'.format(tag))
```

Comment:
trying to minimize the number of lines of logging here, especially since all the tags are added at once
```suggestion
logger.info('Adding the following tags to table: {}'.format(tags))
```